File: 백준/Bronze/32184. 디미고에 가고 싶어！/디미고에 가고 싶어！.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        int dap = 0;
        
        if (A % 2 == 0) {
            dap++;
            A++;
        }
        for (int i = A; i <= B; i += 2) {
            dap++;
        }

        System.out.println(dap);
        sc.close();
    }
}

File: 백준/Bronze/32184. 디미고에 가고 싶어！/디미고에 가고 싶어！.c
#include <stdio.h>

int main() {
    int A, B;
    int dap = 0;
    scanf("%d %d", &A, &B);
    
    if (A % 2 == 0) {
        dap++;
        A++;
    }
    for (int i = A; i <= B; i += 2) {
        dap++;
    }
    
    printf("%d\n", dap);
    return 0;
}
